manifests/module.pp

```diff
@@ -44,11 +53,11 @@
   validate_absolute_path($::selinux::config::module_build_dir)
   validate_absolute_path($::selinux::refpolicy_makefile)
 
-  $module_dir = "${::selinux::config::module_build_dir}/${title}"
+  $module_dir = $::selinux::config::module_build_dir
   $module_file = "${module_dir}/${title}"
 
```

Comment:
`::selinux::config::module_build_dir` != `::selinux::module_build_root` and `::selinux::module_build_dir` does not exist.

you created this var here: https://github.com/voxpupuli/puppet-selinux/pull/189/files#diff-ca85ef38c9b99f20d2d1c8d646a10e77R47 - it confused me too know. maybe we could come up with something more intuitive?  ;-)